client/homebrew/pages/basePages/listPage/brewItem/brewItem.jsx

```diff
@@ -106,7 +106,8 @@ const BrewItem = createClass({
 			<div className='info'>
 				{brew.tags ? <>
```

Comment:
We still have `brew.tags == ['']` to contend with ..

<img width="343" alt="image" src="https://user-images.githubusercontent.com/2605467/188550267-7be37883-d0b2-4d4d-9977-e6470e4760b6.png">

so unless that is getting dealt with soon .. change this test.

Reply:
@G-Ambatte is handling it in a separate PR I believe.

Reply:
Added a temporary filter to remove empty strings from the tags.